test/bin/test-citgm.js

```diff
@@ -30,30 +30,6 @@ test('bin: omg-i-fail /w markdown output /w nodedir', function (t) {
   });
 });
 
-test('bin: omg-i-fail /w custom script /w tap to file /w junit to file /w append', function (t) {
```

Comment:
make sure that we are not losing coverage on some of the options here. We may need to re-add those flags to other tests